src/internal/IOrdinoApp.ts

```diff
@@ -21,7 +21,10 @@ export interface IOrdinoApp {
    * The return value is index in the list of views.
    * @param view ViewWrapper
    */
-  pushImpl(view: ViewWrapper): Promise<number>;
+  pushImpl(view: ViewWrapper);
+
+
+  replaceImpl(previous: ViewWrapper, next:ViewWrapper);
```

Comment:
```suggestion
  /**
   * WRITE ME
   */
  replaceImpl(previous: ViewWrapper, next:ViewWrapper);
```

Function comment is missing.